src/libertem/udf/stddev.py

```diff
@@ -297,12 +313,35 @@ def process_tile(self, tile):
                 varsum_inout=reshaped_view(self.results.varsum, (-1, )),
             )
 
+    def get_results(self):
+        '''
+        Calculate variance, mean and standard deviation from raw UDF results
+
+        Returns
+        -------
+        pass_results : Dict[str, Union[BufferWrapper, numpy.ndarray]]
+        Result dictionary with keys :code:`'sum', 'varsum', 'num_frames'` as :code:`BufferWrapper`,
+            and :code:`'var', 'std', 'mean'` as :class:`numpy.ndarray`.
+        '''
+        num_frames = int(self.results['num_frames'].data[0])
```

Comment:
Probably one should use `.raw_data` instead of `.data`? It works correctly in this case since it is `kind='sig'`, but would set a bad example for `kind='nav'`.

The code is not using the new `self.result()` method, right? This could be a good place for that since the `consolidate_result()` method can make sure that the previous return values are kept for backwards compatibility.

Reply:
> The code is not using the new self.result() method, right? This could be a good place for that since the consolidate_result() method can make sure that the previous return values are kept for backwards compatibility.

That's what I meant with "completely backwards-compatible" - if we change this here, "direct" users of `StdDevUDF` will have to slightly adjust their code. Agree on `.raw_data` for setting an example. Maybe also something for the docs, if I can come up with a reasonable `kind='nav'` example.

Reply:
The original `StdDevUDF` was only returning these buffers as BufferWrapper and the UDF is still returning them exactly like that, right?

```
'varsum': self.buffer(
    kind='sig', dtype=dtype
),
'num_frames': self.buffer(
    kind='single', dtype='int'
),
'sum': self.buffer(
    kind='sig', dtype=dtype
)
```

We are adding new buffers, and I don't think it would make a difference if they are `ndarray` or `BufferWrapper`. We can just make sure that `consolidate_result()`, which is the main "gateway", returns exactly the same as before. Maybe we can deprecate it, although I am unsure if that is worth the effort in this case.